openquake/hazardlib/source/multi_point.py

```diff
@@ -151,19 +152,19 @@ def __fromh5__(self, dic, attrs):
         self.tectonic_region_type = attrs['tectonic_region_type']
         self.magnitude_scaling_relationship = SCALEREL[
             attrs['magnitude_scaling_relationship']]
```

Comment:
Unfortunately `.value` is deprecated, this is why I had to replace it everywhere with `[:]` :-(